third_party/libvirt/AUTHORS

```diff
@@ -0,0 +1,28 @@
+Maintainer
```

Comment:
does generated automatically? if not, can be placed in vender?

Reply:
yes, it is genetated by go-libvirt code-genertor tools, it depends on the version of the libvirtd, but even if the version difference is big, the generated code has little difference. Please refer to https://github.com/digitalocean/go-libvirt.